src/components/Attachment/Unsupported.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import type { Attachment } from 'stream-chat';
+import type { DefaultStreamChatGenerics } from '../../types/types';
+
+export type UnsupportedProps<
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
+> = {
+  attachment: Attachment<StreamChatGenerics>;
+};
+
+export const Unsupported = <
```

Comment:
`import { Unsupported } from 'stream-chat-react'` doesn't say much.
Maybe we should include `Attachment` in the name of this component -> `UnsupportedAttachment` or `UnknownAttachment`